ios/brave-ios/App/iOS/Delegates/AppDelegate.swift

```diff
@@ -86,21 +86,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     SDImageCodersManager.shared.addCoder(PrivateCDNImageCoder())
 
-    // Temporary fix for Bug 1390871 - NSInvalidArgumentException: -[WKContentView menuHelperFindInPage]: unrecognized selector
```

Comment:
No longer needed?

Reply:
Nope, with the introduction of `UIFindInteraction` in iOS 16 the context menu already includes a "find selection" option